File: app/src/main/java/com/teamwork/organizer/ui/projectDetail/fragments/MilestoneFragment.java
package com.teamwork.organizer.ui.projectDetail.fragments;


import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.Menu;
import android.view.MenuInflater;
import android.view.View;
import android.view.ViewGroup;

import com.teamwork.organizer.R;

/**
 * Milestones.
 */
public class MilestoneFragment extends Fragment {


    public MilestoneFragment() {
        // Required empty public constructor
    }


    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setHasOptionsMenu(true);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_milestones, container, false);
    }

    @Override
    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
        inflater.inflate(R.menu.menu_milestone_fragment, menu);
        super.onCreateOptionsMenu(menu, inflater);
    }

}

File: app/src/main/java/com/teamwork/organizer/ui/projectDetail/fragments/MilestoneFragment.kt
package com.teamwork.organizer.ui.projectDetail.fragments


import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.Menu
import android.view.MenuInflater
import android.view.View
import android.view.ViewGroup

import com.teamwork.organizer.R


/**
 * Milestones.
 */
class MilestoneFragment : Fragment() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setHasOptionsMenu(true)
    }

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater!!.inflate(R.layout.fragment_milestones, container, false)
    }

    override fun onCreateOptionsMenu(menu: Menu?, inflater: MenuInflater?) {
        inflater!!.inflate(R.menu.menu_milestone_fragment, menu)
        super.onCreateOptionsMenu(menu, inflater)
    }

}// Required empty public constructor
